src/components/GameOver/GameOver.css

```diff
@@ -0,0 +1,27 @@
+.resultField {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    width: 500px;
```

Comment:
What about resolutions lower than 500px? For those, the game over will go over the screen. and a horizontal scroll will appear.